src/system/dev/vfs.c

```diff
@@ -155,6 +165,64 @@ int _close(int file) {
 	return ret;
 }
 
+int _mkdir(const char* path){
+    struct _reent* r = _REENT;
+	  if (!file_size(file)) {
```

Comment:
Your indentation is off here.

Reply:
This is using spaces, not tabs. You also have a tab and some spaces on some of these lines. The same thing in link.

Reply:
`clang-format` should handle this.